python/pyarrow/tests/test_compute.py

```diff
@@ -574,6 +575,16 @@ def test_binary_slice_compatibility():
         assert expected.equals(result)
         # Positional options
         assert pc.binary_slice(arr, start, stop, step) == result
+        # Fixed size binary input / output
+        for item in data:
+            print(item)
+            print(start, stop, step)
+            fsb_scalar = pa.scalar(item, type=pa.binary(len(item)))
+            expected = item[start:stop:step]
+            print(expected)
```

Comment:
Remove the print statements here?
```suggestion
            fsb_scalar = pa.scalar(item, type=pa.binary(len(item)))
            expected = item[start:stop:step]
```